pkg/controller/agent/pod_test.go

```diff
@@ -720,7 +714,7 @@ fi
 			wantPodSpec: generatePodSpec(func(ps corev1.PodSpec) corev1.PodSpec {
 				ps.Volumes = expectedCAVolume
 				ps.Containers[0].VolumeMounts = expectedCAVolumeMount
-				ps.Containers[0].Command = expectedUbuntuCmd
+				ps.Containers[0].Command = expectedCmd
```

Comment:
There is some duplication in the unit tests now. If you go through the list you will find the same test multiple times once for 7.x and 8.x. The only difference was the different CA script. With a unified script we should remove the duplication in the test cases as those are otherwise just a maintenance burden.

Reply:
The duplicative tests now have been removed.